include/tvm/node/container.h

```diff
@@ -23,17 +23,19 @@
 #ifndef TVM_NODE_CONTAINER_H_
 #define TVM_NODE_CONTAINER_H_
 
+#define TVM_MAP_USE_STL 1
```

Comment:
remove this line eventually

Reply:
Just added a cmake option to turn this off